packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -829,19 +824,21 @@ class VirtualListBasic extends Component {
 	}
 
 	// scrollMode 'translate' only
-	setScrollPosition (x, y, rtl = this.props.rtl, targetX = 0, targetY = 0) {
+	setScrollPositionTarget (x, y) {
+		// The `x`, `y` as parameters in scrollToPosition() are the position when stopping scrolling.
+		// But the `x`, `y` as parameters in setScrollPosition() are the position between current position and the position stopping scrolling.
+		// To know the position when stopping scrolling properly, `x` and `y` are passed and cached in `this.scrollPositionTarget`.
+		if (this.isPrimaryDirectionVertical) {
+			this.scrollPositionTarget = y;
+		} else {
+			this.scrollPositionTarget = x;
```

Comment:
`setScrollPositionTarget` `setScrollPosition` both function are for 'translate'.
I wonder the reason why you divide it to two function?

Reply:
`setScrollPositionTarget` is needed to set the position that a scroller will stop.
` setScrollPosition` is needed to update the scroll position while animating and going to the position that a scroller will stop.

To handle 5-way keys in sandstone VirtualList onAcceleratedKeyDown() properly, we need to know the position that a scroller will stop to make a decision if scrolling into a view or not.

Reply:
I understand. You wanted to omit update the `scrollPositionTarget` in situations such as animations. Please let me know If I misunderstand,